ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -97,9 +98,9 @@ const CancelSpeedupPopover = () => {
     >
       <AppLoadingSpinner className="cancel-speedup-popover__spinner" />
       <div className="cancel-speedup-popover__wrapper">
-        <Typography
+        <Text
           boxProps={{ alignItems: AlignItems.center, display: DISPLAY.FLEX }}
-          variant={TypographyVariant.H6}
+          variant={TextVariant.bodySm} as="h6"
```

Comment:
Update 
```
 <Text
          boxProps={{ alignItems: AlignItems.center, display: DISPLAY.FLEX }}
          variant={TextVariant.bodySm} as="h6"
```

to 
```
 <Text
          alignItems={AlignItems.center}
          display: {DISPLAY.FLEX }
          variant={TextVariant.bodySm} as="h6"
```

Reply:
Hey @NidhiKJha 
Its supposed to be ```display= {DISPLAY.FLEX}``` instead of ```display: {DISPLAY.FLEX}```, right? because the code throws an error otherwise.
Also, I coulnt find any screenshot of ```cancel-speed-popover``` in the storybook components thats why I was not able to add it in the before/after

Reply:
Ah! yes, It should be `display= {DISPLAY.FLEX}`. Thanks for updating it

Reply:
made the changes but it failed a few tests due to formatting issues, will recommit now.